src/test/java/com/wikia/webdriver/common/core/Assertion.java

```diff
@@ -60,19 +60,26 @@ public static boolean assertStringNotContains(String current, String pattern) {
   public static void assertEquals(String current, String pattern) {
     String patternEncoded = encodeSpecialChars(pattern);
     String currentEncoded = encodeSpecialChars(current);
+    AssertionError caughtException = null;
     boolean assertion = true;
     try {
       Assert.assertEquals(current, pattern);
     } catch (AssertionError err) {
       addVerificationFailure(err);
       assertion = false;
+      caughtException = err;
+    }
+    //TODO: Check that!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)